numpy/_core/tests/test_numeric.py

```diff
@@ -2739,11 +2739,6 @@ def test_object_clip(self):
         expected = np.array([1, 1, 2, 3, 4, 5, 5, 5, 5, 5])
         assert actual.tolist() == expected.tolist()
 
-    def test_clip_all_none(self):
-        a = np.arange(10, dtype=object)
-        with assert_raises_regex(ValueError, 'max or min'):
```

Comment:
Rather than delete the test, keep it and check that it works?
```
result = np.clip(a, None, None)
assert result is a
result2 = np.clip(a)
assert result2 is a
```

Reply:
Done!